### ISSUE TITLE
Incorrect conversion

### ISSUE THREAD

**Issue by RocketFlash** [yzhou377/argoverse-kitti-adapter#2]
Hi, @yzhou377 ! Thank you for your work on argo to kitti converter! 
Using this script, my data was not converted correctly
Am I doing something wrong or is there a mistake in the code?

<img width="681" alt="Screenshot 2020-04-07 at 09 11 44" src="https://user-images.githubusercontent.com/7099937/78636413-f64c6380-78b0-11ea-9e75-c295233a82fd.png">

**Comment by rvalienter90** [comment]
Hi, @yzhou377 ! Thank you the great work!
I am also facing the same error with the GT labels, when I got the predictions using P-RCNN the 3D BBs are correct, but for the adapter.py output the ground truth 3D BBs are not right,  2D BBs are ok. **Could you please help with that?**
In the image GT is green, predictions are in red.3D BB for ground truth are wrong (shifted up) 2D BB are ok 
![capture](https://user-images.githubusercontent.com/26827842/86545547-7bf9a280-bee4-11ea-808a-a892c57d25e2.PNG)

Also Kitti dataset have stereo images, Do you plan to support Argoverse to Kitti stereo and sync?
Thanks!

**Comment by harishrithish7** [comment]
Hi @RocketFlash @rvalienter90 ,
Have you guys fixed this?

**Comment by RocketFlash** [comment]
Yes, the problem was in visualization, or in data representation. Conversion code from this repo presents box as center point of the box, but kitti uses center of the bottom of the box format. My advice is to try correctly understand data representations.

**Comment by harishrithish7** [comment]
Thank you for the quick response. Works for me now!
But there still seems to be a few other issues, 2d bounding box is not correct for all objects and even orientation.
Have you noticed any other issues?

**Comment by issouker97** [comment]
> Thank you for the quick response. Works for me now! But there still seems to be a few other issues, 2d bounding box is not correct for all objects and even orientation. Have you noticed any other issues?

Hey, did you manage to solve the problems found before, have you got at the end a good conversion from argoverse to kitti, if yes please, can you help me !! i am working on acdemic project about this
